charts/newrelic-logging/README.md

```diff
@@ -126,6 +126,7 @@ See [values.yaml](values.yaml) for the default values
 | `image.pullPolicy`                                         | The pull policy.                                                                                                                                                                                                                                                             | `IfNotPresent`                                                                  |
 | `image.pullSecrets`                                        | Image pull secrets.                                                                                                                                                                                                                                                          | `nil`                                                                           |
 | `image.tag`                                                | The version of the container to pull.                                                                                                                                                                                                                                        | See value in [values.yaml]`                                                     |
+| `ports`                                                    | Any ports you wish to expose from the pod.  Ex. 2020 for metrics                                                                                                                                                                                                             | `[]`                                                                            |
```

Comment:
Or even, I was thinking on making this a boolean called `exposeMetrics`, which internally results in exposing the 2020 port that Fluent Bit uses. I believe it could be more user friendly. What do you think?

Reply:
@jsubirat - yes agree that an `exposeMetrics` would be useful.  If you can accept this PR then I can add that functionality in a separate PR.   The ability to expose ports may be useful outside of any default prometheus implementation.